recipes/ncrf/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+mkdir -p $PREFIX/bin
+
+make 
+
+chmod +x NCRF
+
+mv NCRF $PREFIX/bin
+
+chmod +x *.py
+cp *.py $PREFIX/bin/
+cp ncrf_to_bed.py $PREFIX/bin/ncrf_to_bed
```

Comment:
you copied them already above, should this maybe a symlink?

Reply:
Thanks.  I had tried using symlinks instead of all the copies, but in my local test phase that failed. I was unable to figure out why (I am a newbie to bioconda), so I resorted to doing all the copies.

So, e.g. "ln -s ncrf_to_bed.py $PREFIX/bin/ncrf_to_bed" ought to work, right?

Reply:
I'm not clear how to proceed.  Should I (a) cancel this PR, (b) change my second copies to symlinks, (c) commit, and (d) do another PR?

Reply:
@rsharris I think (b) is what @bgruening is asking for. Make a commit that changes the extra copies to symlinking and push it to `makovalab-psu:ncrf`, this PR will automatically update.